wordpress-mobile/WordPress-Android
# Issue/3007 reader detail bump stats

Resolves #3007 - viewing a post in reader detail now bumps the pageview for that post in the author's stats.

To test:
* Comment out [this code](https://github.com/wordpress-mobile/WordPress-Android/blob/issue/3007-reader-detail-bump-stats/WordPress/src/main/java/org/wordpress/android/ui/reader/actions/ReaderPostActions.java#L256-L259) which prevents bumping stats for posts is blogs the current user is an admin of
* Follow one of your test blogs and add a new post to the blog
* Open the reader and view Blogs I Follow
* Tap your new post in the list to open the detail view
* Wait a few minutes
* Go to your stats and confirm that the view is showing up

Note that you should also see "WordPress.com Reader" in the referrers.
